keyboards/ergodox_ez/keymaps/stamm/config.h

```diff
@@ -0,0 +1,23 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+/* https://docs.qmk.fm/#/tap_hold */
+#define ORYX_CONFIGURATOR
```

Comment:
This needs/should have an include guard at the top:
```suggestion
#pragma once

#define ORYX_CONFIGURATOR
```